docs/integrations/twilio/webhooks.md

```diff
@@ -2,120 +2,155 @@
+    ![Twilio URL to Publish](img/ngrok_url_configuration_twilio.png)
+
+1. Click **Save**.
+
+1. Select your team for the **TEAM** field, click the **created** checkbox for **Projects** under the **EVENTS** section, and then click **Submit**.
 
-Congrats, everything is configured! Now it's time to test.
 
 ### Run Webhooks with Twilio and ngrok
 
-1. Send an SMS message to your Twilio Phone number that was configured in the steps above. 
-2. Get excited (and maybe a little scared) when you get a text back saying, **"The Robots are coming! Head for the hills!"**
+Twilio sends different request body contents depending on the event that is being triggered.
+You can trigger new calls from Twilio to your application by following the instructions below.
 
-Congrats! You got an end-to-end example working but there's even more you can do with ngrok that will make development even easier. Check out how to inspect and replay your requests without having to send an SMS. Trust me, you won't regret it.
+1. Send an SMS message to the Twilio Phone number that you configured in the previous steps.
 
-## Optional next steps {#optional}
+    Confirm your localhost app receives a notification and logs both headers and body in the terminal.
 
-### Add additional security using ngrok's signature webhook verification
-The webhook verification module allows ngrok to assert requests to your endpoint originate from Twillio. This is a quick step to add extra protection to your setup. 
+Optionally, you can verify the log of the webhook call in Twilio:
 
-**Note:** This ngrok feature is limited to 500 validations per month on free ngrok accounts. For unlimited, upgrade to Pro or Enterprise.
+1. On the your phone page, click the **Messages Log** tab, select **Incoming** in the **Direction** field, and then click **Filter**.
+    ![Webhook Logs](img/ngrok_logs_twilio.png)
 
-1. Login to [Twilio Console](https://console.twilio.com/) and copy your Auth Token value.<br />
-    **Note:** ngrok Webhook Verification ensures traffic from your Twilio account is the **only traffic allowed** to make calls to your app. Because Twilio signs all Webhooks using the Primary Auth Token, ngrok can verifies the signature of every request and only authorizing requests originating from your Twilio account. 
-2. Restart ngrok by running the command, replacing {your auth token} with your Twilio Auth Token:
-    ```bash
-    ngrok http 3000 --verify-webhook twilio --verify-webhook-secret {your auth token}
-    ```
 
 ### Inspecting requests
 
-When you launch ngrok, you can see two links: One for the tunnel to your app (it ends up in ngrok.io unless you're using custom domains) and a local URL (http://localhost:4040) for the Request Inspector.
+When you launch the ngrok agent on your local machine, you can see two links: one for the tunnel to your app (it ends up in `ngrok.io` unless you're using custom domains) and a local URL for the Web Interface (a.k.a **Request Inspector**).
```

Comment:
We should change the wording here since this has changed with the new Global Network domains. 

> ends with `ngrok-free.app` for free accounts or `ngrok.app` for paid accounts when not using custom domains.

Feels long though.

Reply:
I like this way.
Should we name the first like as Forwarding as this is what shows in the terminal?
If so, I can create  a PR to fix for all the others.

Reply:
Is it better?

When you launch the ngrok agent on your local machine, you can see two links: 
* The URL to your app (it ends with `ngrok-free.app` for free accounts or `ngrok.app` for paid accounts when not using custom domains).
* A local URL for the Web Interface (a.k.a **Request Inspector**).